ydb/docs/en/core/reference/ydb-sdk/topic.md

```diff
@@ -1382,6 +1382,8 @@ If a commit fails with an error, the application should log it and continue; it
    }
    ```
 
+   The `Commit` call is fast by default, saving data into an internal buffer and returning control back to the caller. The real message to the server is sent in the background. To prevent losing the last commits, you should call the `Reader.Close()` method before exiting the program.
```

Comment:
Let's also explain what people are supposed to do to guarantee that messages are not lost even if the client is abruptly terminated due to OOM, segfault, etc., and can't catch it to close the connection explicitly.

Reply:
It is reader description and messes stored on a server already. If a process will be killed before commit a message - the message will resend to other reader by server. The client does not have to do anything for this.